### PR TITLE
Add workflow frameworks and github.com/luno/workflow

### PR DESCRIPTION
**Pull request by andrewwormald**
## We want to ensure high quality of the packages. Make sure that you've checked the boxes below before sending a pull request.

- [x] I have read the [Contribution Guidelines](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md#contribution-guidelines)
- [x] I have read the [Maintainers Note](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md#maintainers)
- [x] I have read the [Quality Standards](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md#quality-standards)

_Not every repository (project) will require every option, but most projects should. Check the Contribution Guidelines for details._

- [x] The repo documentation has a pkg.go.dev link.
- [x] The repo documentation has a coverage service link.
- [x] The repo documentation has a goreportcard link.
- [x] The repo has a version-numbered release and a go.mod file.
- [x] The repo has a continuous integration process that automatically runs tests that must pass before new pull requests are merged.
- [x] Continuous integration is used to attempt to catch issues prior to releasing this package to end-users.

## Please provide some links to your package to ease the review

- [x] forge link (github.com, gitlab.com, etc): https://github.com/luno/workflow
- [x] pkg.go.dev: https://pkg.go.dev/github.com/luno/workflow
- [x] goreportcard.com: https://goreportcard.com/report/github.com/luno/workflow
- [x] coverage service link ([codecov](https://codecov.io/), [coveralls](https://coveralls.io/), etc.): https://sonarcloud.io/summary/new_code?id=luno_workflow

## Pull Request content

- [x] The package has been added to the list in alphabetical order.
- [x] The package has an appropriate description with correct grammar.
- [x] As far as I know, the package has not been listed here before.

## Category quality

_Note that new categories can be added only when there are 3 packages or more._

Packages added a long time ago might not meet the current guidelines anymore. It would be very helpful if you could check 3-5 packages above and below your submission to ensure that they also still meet the Quality Standards.

Please delete one of the following lines:

- [x] The packages around my addition still meet the Quality Standards.
- [x] I removed the following packages around my addition: (please give a short reason for each removal)

https://github.com/albrow/jobs - archived

Thanks for your PR, you're awesome! :sunglasses:

### CODE REVIEW

**Comment by yassinebenaid** [comment]
You mentioned that you've removed: https://github.com/albrow/jobs. I think you forget to remove it as it doesn't show up as removed in your changes.

**Review by melladoo — APPROVED** [review]

**Review by yassinebenaid — CHANGES_REQUESTED** [review]
1. You have added a new category, make sure you list it upon categories list.
2. Make sure the category is in the right order, `Workflow` cannot come before `JSON`. the same goes for categories list on top of the README.
3. You mentioned that you've removed the [`https://github.com/albrow/jobs`](https://github.com/albrow/jobs), it doesn't seem to be removed in your changes.

**Comment by andrewwormald** [comment]
> 1. You have added a new category, make sure you list it upon categories list.
> 2. Make sure the category is in the right order, `Workflow` cannot come before `JSON`. the same goes for categories list on top of the README.
> 3. You mentioned that you've removed the [`https://github.com/albrow/jobs`](https://github.com/albrow/jobs), it doesn't seem to be removed in your changes.

Thanks @yassinebenaid, will make the appropriate changes.

**Comment by andrewwormald** [comment]
@yassinebenaid found the removal of `https://github.com/albrow/jobs` in a unpushed commit. Should be all good now.

**Review by yassinebenaid — APPROVED** [review]
Order was wrong, I've fixed it.

Thanks for valuable contribution.

**Comment by andrewwormald** [comment]
Brilliant - thank you

File: README.md [hunk 1]
```diff
@@ -1492,22 +1493,17 @@ _Libraries for programming devices of the IoT._
 ## Job Scheduler
 
 _Libraries for scheduling jobs._
-- [Cadence-client](https://github.com/uber-go/cadence-client) - A framework for authoring workflows and activities running on top of the Cadence orchestration engine made by Uber.
```
Comment (Israelgladys) [inline README.md:1495]: --An action needed on this

Comment (yassinebenaid) [reply README.md:1495]: would you give us more context on what action are you referring to!

**Pull request merged**